Cargo.toml

```diff
@@ -12,4 +12,4 @@ http = []
 [dependencies]
 smol = "1.2.5"
 json = "0.12.4"
-
+tempfile = "3"
```

Comment:
why we need tempfile as a dependency? the std::fs seems to be enough

Reply:
I try it